File: C_C++/program13.c
// find sum of even and odd digits

#include<stdio.h>

int main()
{
    int a;
    int rem;
    int even=0,odd=0;
    printf("Enter the number:");
    scanf("%d",&a);

    while(a!=0)
    {
        rem=a%10;
        if(rem%2==0)
        {
            even+=rem;
        }
        else
        {
            odd+=rem;
        }
        a=a/10;
    }
    printf("Even sum:%d \n Odd sum:%d",even,odd);
}

File: Python/program13.py
#find sum of even and odd digits

num=int(input())
sum_even=0
sum_odd=0

while(num>0):
    rem=num%10
    if(rem%2==0):
        sum_even+=rem
    else:
        sum_odd+=rem
    num//=10

print(f"Sum of even numbers:{sum_even} and sum of odd numbers:{sum_odd}")

